app/controllers/application_controller.rb

```diff
@@ -9,6 +9,8 @@ class ApplicationController < ActionController::Base
   # Ensure every controller authorizes resource or skips authorization (skip_authorization_check)
   check_authorization unless: :devise_controller?
 
+  before_action :devise_configure_permitted_parameters, if: :devise_controller?
```

Comment:
devise_parameter_sanitizer.permit(:accept_invitation, keys: [:username])

used to add username field in the registration page for invited users otherwise only password and password confirmation fields were present.